### ISSUE TITLE
Parse Error : syntax error, unexpected 'extends' (T_EXTENDS), expecting T_STRING on line 12  10 |  11 | class  > 12 | extends Model   13 | {  14 |  15 | use HasFactory;

### ISSUE THREAD

**Issue by ocontreras4** [VemtoOrg/vemto2-issues#126]
**Error Message:**
Parse Error : syntax error, unexpected 'extends' (T_EXTENDS), expecting T_STRING on line 12
  10 |
  11 | class  
> 12 | extends Model 
  13 | {
  14 |
  15 | use HasFactory;

**Stack:**
```
SyntaxError: Parse Error : syntax error, unexpected 'extends' (T_EXTENDS), expecting T_STRING on line 12
  10 |
  11 | class  
> 12 | extends Model 
  13 | {
  14 |
  15 | use HasFactory;
    at mt.raiseError (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:284:175)
    at mt.error (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:284:830)
    at mt.expect (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:285:414)
    at mt.read_class_declaration_statement (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:273:4184)
    at mt.read_top_statement (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:280:1244)
    at mt.read_top_statements (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:280:907)
    at mt.read_namespace (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:273:36852)
    at mt.read_start (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:273:36534)
    at mt.parse (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:283:15386)
    at td.parseCode (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.1c142fe5.js:286:2410)
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.10
win32 10.0.22631
Locale: en-US
App version: 2.0.10

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.11. @ocontreras4 please check if you have a model without a name on your Schema definition

**Comment by ocontreras4** [comment]
Hi, Mr. Thiago before I Update to version 2.11 since your ask if one of my models has an empty name ! I check if by clicking the Sync Botton save the models's name and it works ! maybe I was missing this step before I tried to save the schema changes ! any way it looks like there is a Bug because it should say "Sync" Before saving changes !
When O generate the Code, it gave me 3 problems That I solved with the option AI it seems to work until now !
Thanks for your quick support!

Atte. Oscar Contreras
________________________________
